lib/services/tokens/token.dao.js

```diff
@@ -57,7 +54,7 @@ dao.find = function (tokenObj, options) {
 
   return db.hgetallAsync(consumerTokensKey)
     .then((tokenIds) => {
-      let tokenPromises, activeTokenIds, getTokenPromise;
+      let activeTokenIds;
```

Comment:
We need a style guide, or we will keep changing these lines of code according to the tastes of the person that's working on it.

Reply:
I didn't want to change logic / style of existing code. I only removed `bluebird` promise library.